packages/web/pages/api/experience/assets/[asset].ts

```diff
@@ -0,0 +1,25 @@
+import { NextApiRequest, NextApiResponse } from 'next'
```

Comment:
why does this filename has brackets in it? Is that a Next.JS thing?

Reply:
yeah its a way of doing what in express would be `/assets/:asset` essentially a url param